src/algorithms/uncategorized/kadane's algorithm/kadane_algo.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Simplest Kadane's algorithm implementation.
+ *
+ * @param {*[]} array
+ * @return {number[]}
+ */
+
+export default function kadane() {
+  let current_sum = array[0];
```

Comment:
what is `array`?

Reply:
Thanks for pointing out , i forgot to add parameter to function.